packages/docusaurus-theme-redoc/src/theme/Redoc/styles.css

```diff
@@ -71,18 +71,14 @@ html:not([data-theme="dark"]) .redocusaurus div[id^="react-tabs"] > div:nth-chil
 
 /* ------ Schema Styling Overrides ------- */
 
-.redocusaurus table th, table td {
+.redocusaurus table th, .redocusaurus table td {
   border: none;
 }
 
-.redocusaurus table:not(.security-details) td {
-  border-bottom: none !important;
-}
-
-.redocusaurus table.security-details > tbody > tr  {
+.redocusaurus table > tbody > tr  {
   color: var(--ifm-font-color);
```

Comment:
I just realized, `--ifm-font-color` is also undefined, maybe this should be changed to `--ifm-font-color-base`?